__tests__/integration.test.js

```diff
@@ -114,3 +114,42 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with an array of comments with the correct article_id sorted in descending order", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(2);
```

Comment:
It would be preferable to use an article that has more that 2 comments for confidence that they are actually sorted